core/src/main/resources/operatorsScore.csv

```diff
@@ -250,9 +250,15 @@ WeekDay,4
 WindowExpression,4
 WindowSpecDefinition,4
 Year,4
-KMeans,8.86
-PCA,2.24
-LinearRegression,2
-RandomForestClassifier,6.31
-RandomForestRegressor,3.66
-XGBoost,4.45
+KMeans-pyspark,8.86
+KMeans-scala,1
+PCA-pyspark,2.24
+PCA-scala,2.69
+LinearRegression-pyspark,2
+LinearRegression-scala,1
+RandomForestClassifier-pyspark,6.31
+RandomForestClassifier-scala,1
+RandomForestRegressor-pyspark,3.66
+RandomForestRegressor-scala,1
+XGBoost-pyspark,1
+XGBoost-scala,3.31
```

Comment:
Just to confirm, the ML functions in scala (and XGBoost in pyspark) have zero speedup?  Or we just don't have results yet so we're leaving them defaulted to 1.